From 9b72a86292fb2c34d7be7b928ac06f2609de0f43 Mon Sep 17 00:00:00 2001
From: Thomas Huth <thuth@redhat.com>
Date: Thu, 30 Sep 2021 09:25:23 +0200
Subject: [PATCH 2/2] redhat: Add s390x machine type compatibility update for
 6.1 rebase
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Thomas Huth <thuth@redhat.com>
RH-MergeRequest: 45: Add s390x machine type compatibility update for 6.1 rebase
RH-Commit: [2/2] 6f71801ac2d77b82b010eac46cd82cd74c53f246 (thuth/qemu-kvm-cs9)
RH-Bugzilla: 1998943
RH-Acked-by: Greg Kurz <gkurz@redhat.com>
RH-Acked-by: Daniel P. Berrangé <berrange@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: David Hildenbrand <david@redhat.com>

Add hw_compat_rhel_8_5 for the s390-ccw-virtio-rhel8.5.0 machine
type. It's currently not urgently required, since hw_compat_rhel_8_5
only contains entries that are not related to s390x, but just in
case some other entries are added there later, it's better if we
add this right from the start to the s390x machine, too.

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1998943
Signed-off-by: Thomas Huth <thuth@redhat.com>
---
 hw/s390x/s390-virtio-ccw.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/s390x/s390-virtio-ccw.c b/hw/s390x/s390-virtio-ccw.c
index 9a51dd8de2..4ee8d266ec 100644
--- a/hw/s390x/s390-virtio-ccw.c
+++ b/hw/s390x/s390-virtio-ccw.c
@@ -1091,6 +1091,7 @@ static void ccw_machine_rhel850_instance_options(MachineState *machine)
 
 static void ccw_machine_rhel850_class_options(MachineClass *mc)
 {
+    compat_props_add(mc->compat_props, hw_compat_rhel_8_5, hw_compat_rhel_8_5_len);
 }
 DEFINE_CCW_MACHINE(rhel850, "rhel8.5.0", true);
 
-- 
2.27.0

